test/mock-components/mock-business-address.js

```diff
@@ -0,0 +1 @@
+module.exports = '25 test Farm Road, BS33 0NP'

```

Comment:
objects are separated so that they can easily be re-used without too much overload, putting too many object in the  same file may become clumsy by the time we have more objects.

Reply:
Fine by me!